httpclient/logging_round_tripper.go

```diff
@@ -96,6 +96,7 @@ func (rt *LoggingRoundTripper) getLogger(ctx context.Context) log.FieldLogger {
 }
 
 // RoundTrip adds logging capabilities to the HTTP transport.
+// Notice - external_request time is cumulative and will not be accurate if multiple concurrent requests are done
```

Comment:
Could you please elaborate on what the external_request time represents and clarify its connection to middleware.LoggingParams? This additional context would help clarify its purpose.